zig-mode.el

```diff
@@ -158,7 +158,9 @@ If given a SOURCE, execute the CMD on it."
                  (compilation-mode)
                  (when zig-return-to-buffer-after-format
                    (pop-to-buffer file-buffer))))
-           (revert-buffer :ignore-auto :noconfirm)))))))
+           (let ((active-minor-modes-list minor-mode-list))
+	     (revert-buffer :ignore-auto :noconfirm :preserve-modes)
+	     (setq minor-mode-list active-minor-modes-list))))))))
```

Comment:
This is nitpicky, but could you indent this with `indent-tabs-mode` off? It helps keep the indentation of this source consistent.

See also https://github.com/ziglang/zig-mode/pull/65#discussion_r775442202